controllers/topics/getLatestPost.js

```diff
@@ -0,0 +1,52 @@
+const {QueryTypes} = require('sequelize');
+const {sequelize, Topics} = require('../../databases/models');
+
+module.exports = async (req, res) => {
+  const {name} = req.query;
+  try {
+    let topics = await Topics.findOne({
+      where: {
+        name: name
+      }
+    });
+
+    if (!topics) {
+      res.status(400).json({
+        code: 400,
+        message: 'Topic not found'
+      });
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

If the topic is not found, the function continues to execute and will eventually fail when trying to access `topics.topic_id`. You should add a `return` statement after sending the response to stop further execution.



<!-- This is an auto-generated comment by OSS CodeRabbit -->